Modules/Exercise05/Tests/Unit/Services/OrderServiceTest.php

```diff
@@ -0,0 +1,130 @@
+<?php
+
+namespace Tests\Unit\Services;
+
+use Carbon\Carbon;
+use Modules\Exercise05\Services\OrderService;
+use Tests\TestCase;
+
+class OrderServiceTest extends TestCase
+{
+    protected $calendarServiceMock;
+
+    public function setUp(): void
+    {
+        parent::setUp();
+        $this->service = new OrderService();
+    }
+
+    /**
+     * @dataProvider provider_test_handle_discount
+     */
+    public function test_handle_discount($detailOrder, $discount)
+    {
+        $result = $this->service->handleDiscount($detailOrder);
+
+        $this->assertEquals($discount, $result);
+    }
+
+    public function provider_test_handle_discount()
```

Comment:
Trong provider này nên thêm có 1 case nữa check độ chính xác tới 2 chữ số thập phân vì cái round nó có thêm tham số thứ 2 là `2`
```
$infoBill['price'] = round(($detailOrder['price'] * 80) / 100, 2);
```
Cái này nice to have :smile: